fuel/transformers/__init__.py

```diff
@@ -60,6 +61,24 @@ def get_epoch_iterator(self, **kwargs):
         self.child_epoch_iterator = self.data_stream.get_epoch_iterator()
         return super(Transformer, self).get_epoch_iterator(**kwargs)
 
+    def get_data(self, request=None):
+        if self.batch_input:
+            return self.get_data_from_batch(request)
+        else:
+            return self.get_data_from_example(request)
+
+    def get_data_from_example(self, request=None):
```

Comment:
It's really exactly the same as it was! It's just that `get_batch` has been renamed to `get_data_from_batch` to make clear that `get_batch` doesn't *return* a batch, but that it expects a batch as input. Another naming suggestion is welcome, but I find things like `get_from_batch`, `from_batch` unclear.